Lib/platform.py

```diff
@@ -197,7 +197,7 @@ def libc_ver(executable=None, lib='', version='', chunksize=16384):
         | (GLIBC_([0-9.]+))
         | (libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)
         | (musl-([0-9.]+))
-        | (libc.musl(?:-\w+)?.so(?:\.(\d[0-9.]*))?)
+        | ((?:libc.|ld-)musl(?:-\w+)?.so(?:\.(\d[0-9.]*))?)
```

Comment:
There is a bug in the existing regex, it should be `libc\.` instead of `libc.`, no?

```suggestion
        | ((?:libc\.|ld-)musl(?:-\w+)?.so(?:\.(\d[0-9.]*))?)
```

Reply:
I saw the bug and thought about fixing it, but then I wondered what esoteric musl-based distribution would break if the regex gets more specific. If this is not a concern, we can quote the dot.

Reply:
If later we discover Linux distributions where musl is not properly detected, we can update the regex again. For now, I would prefer to use a strict `libc\.`.

Reply:
Done.